### DISCUSSION TITLE
Triple RPS Boost Update to Maximise dApp Performance in the sui Bull Market

### DISCUSSION THREAD

**Discussion by 0x0factor** [0x0factor/python_blockchain_app-master#2] [Q&A] [answered]
Dear all Sui Devs,

Please be informed of a technical update on GetBlock RPC Node Provider. The rate limit for all shared plans of Sui Nodes will be increased from 60 to 200 requests/second (RPS). This update is part of ongoing service enhancements.
It is important to note that this RPS increase applies solely to users who have migrated to access tokens. Users currently employing API keys and wishing to benefit from the higher rate limit must migrate to access tokens.
For detailed instructions on the migration process, refer to the instructions available in your account.

Regards,
Oyatech Technical Team

**Comment by goldapples** [comment, marked as answer]
thank for your team's effort
